docs/_includes/calendar.html

```diff
@@ -0,0 +1,396 @@
+                    <div class="calendar__month">
+                        {% include calendar-month.html index="3" %}
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+    {% highlight html %}
+<div class="calendar">
+    <div class="calendar__body">
+        <div class="calendar__month">
+            <table>
+                <caption>
+                    August 2019
+                </caption>
+                <thead>
+                    <tr>
+                        <th>Su</th><th>Mo</th><th>Tu</th><th>We</th><th>Th</th><th>Fr</th><th>Sa</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <td colspan="2"></td>
+                        <td><span>1</span></td>
+                        <td><span>2</span></td>
+                        <td><span>3</span></td>
+                        <td><span>4</span></td>
+                        <td><span>5</span></td>
+                    </tr>
+                    <tr>
+                        <td><span>6</span></td>
+                        <td><span>7</span></td>
+                        <td><span>8</span></td>
+                        <td><span>9</span></td>
+                        <td><span>10</span></td>
+                        <td><span>11</span></td>
+                        <td><span>12</span></td>
+                    </tr>
+                    ...
+                    <tr>
+                        <td><span>27</span></td>
+                        <td><span>28</span></td>
+                        <td><span>29</span></td>
+                        <td><span>30</span></td>
+                        <td><span>31</span></td>
+                    </tr>
+                </tbody>
+            </table>
+        </div>
+    </div>
+</div>
+    {% endhighlight %}
+
+    <h3>Calendar with Disabled Cells and "Today"</h3>
+
+    <p>Each cell can be disabled, and the current day can be marked with aria roles.</p>
+
+    <div class="demo">
+        <div class="demo__inner">
+            <div class="calendar">
+                <div class="calendar__body">
+                    <div class="calendar__month">
+                        {% include calendar-month.html index="1" %}
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+    {% highlight html %}
+<div class="calendar">
+    <div class="calendar__body">
+        <div class="calendar__month">
+            <table>
+                <caption>
+                    August 2019
+                </caption>
+                <thead>
+                    <tr>
+                        <th>Su</th><th>Mo</th><th>Tu</th><th>We</th><th>Th</th><th>Fr</th><th>Sa</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <td colspan="4"></td>
+                        <td><span class="calendar__cell--disabled">1</span></td>
```

Comment:
It seems like we are conveying _meaning_ using color alone here (via the grey text). We may need to do something like:

```html
<td><span class="calendar__cell--disabled">3<span class="clipped">3 - past date</span></span></td>
```

or

```html
<td><span class="calendar__cell--disabled" aria-label="3 - past date">3</span></td>
```

Reply:
Good catch. The specific message probably needs to change based on the specific implementation, but I should add an `aria-label` for spans in general.